app/src/common/urls.js

```diff
@@ -173,10 +173,17 @@ export const URLS = {
       'filter.in.level': 'ERROR',
       'page.sort': 'logTime,DESC',
     })}`,
-  logItemStackTraceMessageLocation: (activeProject, itemId, stackTraceItemId, pageSize, level) =>
+  logItemStackTraceMessageLocation: (
+    activeProject,
+    itemId,
+    stackTraceItemId,
+    pageSize,
+    pagePage,
```

Comment:
Please also fix usage in `app/src/pages/inside/logsPage/logItemInfo/logItemInfoTabs/stackTrace/stackTrace.jsx#160`